### PR TITLE
remove duplicated code, pull in ad2games-theme as dependency instead

### PR DESCRIPTION
**Pull request by matthewcorreia**

### CODE REVIEW

File: lib/ui_components/version.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module UiComponents
-  VERSION = '2.2.1'
+  VERSION = '2.2.2'
```
Comment (RKushnir) [inline lib/ui_components/version.rb]: I would increment the minor version(i.e. `2.3.0`), seems like a bigger change.

Comment (tsujigiri) [inline lib/ui_components/version.rb]: Does it change anything in terms of features/how to use it?

Comment (RKushnir) [inline lib/ui_components/version.rb]: It adds a new dependency, which may break something. For your consideration.

Comment (matthewcorreia) [inline lib/ui_components/version.rb]: @tsujigiri shouldn't change anything about how to use it.
@RKushnir i will increase the minor version anyway - it's true that it's a fairly significant change.

**Pull request merged**